README.md

```diff
@@ -69,7 +70,7 @@ Try the steps below to get a feel for how quick the setup is:
 2. Follow the instructions detailed on our
    [documentation page][migrate-from-stripe].
```

Comment:
Could you please fix the numbering?

Reply:
What do you mean?

Reply:
My bad, ignore. I thought the numbering was 1, 3, 2 because of the way the diff was displayed.